### PR TITLE
Xedra Spy profession, mission, safehouse and fixes

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Xedra: Spy profession, mission, location and fixes"

#### Purpose of change
Adding more stuff while I'm on a roll

#### Describe the solution
Adds items, profession, mission, location.  Might add some scenarios.

#### Describe alternatives you've considered
None

#### Testing
Tested and it looked good so far.

### CODE REVIEW

File: data/mods/Xedra_Evolved/player/professions.json [hunk 1]
```diff
@@ -50,5 +50,22 @@
       "male": [ "briefs" ],
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "spy",
+    "name": "Foreign Agent",
+    "description": "You came here from another country trying to gain access to secrets the United States had ripped from other worlds.  They probably had sent people to your homeland to do the same.  You were out performing recon when the Cataclysm struck and caught a ride with a stranger trying to find safety.  The car must have wrecked and he died a few days back but you aren't sure where you are and you only have a vague idea of how to get back to the safehouse, where you have some more gear stowed.",
```
Comment (Termineitor244) [inline data/mods/Xedra_Evolved/player/professions.json:58]:
```suggestion
    "description": "You came here from another country trying to gain access to secrets the United States had ripped from other worlds.  They probably had sent people to your homeland to do the same.  You were out performing recon when the Cataclysm struck and caught a ride with a stranger trying to find safety.  Sadly you lost your companion and vehicle, getting lost in the process and with now only a vague idea on how to get back to the safehouse, where you hope your hidden gear remains untouched.",
```

**Review by Termineitor244 — COMMENTED** [review]
Maybe this is better worded? And this way you can add the profession to the helicopter crash too since it remains vague what happened to your vehicle and companion (Allowing more diverse roleplaying for it).

**Comment by Maleclypse** [comment]
> Maybe this is better worded? And this way you can add the profession to the helicopter crash too since it remains vague what happened to your vehicle and companion (Allowing more diverse roleplaying for it).

Good call.

**Comment by Maleclypse** [comment]
@Termineitor244 do you see what's wrong with the new itemgroup for the spektor?  It seems like when that group spawns there aren't any bullets in it despite me specifying the magazines containing ammo.

**Comment by Termineitor244** [comment]
You beat me to it xD

**Comment by Maleclypse** [comment]
@TheShadowFerret I made guesses at the 9x21mm damage and armor piercing based on the wikipedia article and the gun damage guide in game_balance.

**Comment by TheShadowFerret** [comment]
> @TheShadowFerret I made guesses at the 9x21mm damage and armor piercing based on the wikipedia article and the gun damage guide in game_balance.

The sources Wikipedia cite say it's either [420](https://web.archive.org/web/20070928171038/http://www.modernfirearms.net/handguns/hg25-e.htm) ([other source](https://web.archive.org/web/20140107225252/http://gunsru.ru/rg_patron_9x21_eng.html) they use) or [430](https://web.archive.org/web/20070928170953/http://www.modernfirearms.net/ammo/sp-e.htm#SP-10) m/s, but that shouldn't really matter :P

**Pull request merged**